jsx/PaginationLinks.js

```diff
@@ -1,148 +1,135 @@
-/* exported RPaginationLinks */
+import React, {useState} from 'react';
 
-import React, {Component} from 'react';
-import PropTypes from 'prop-types';
+const theme = {
+  primary: '#053665',
+  secondary: '#A6D3F5',
+  accent: '#E89A0C',
+  neutral: '#DDD',
+};
 
 /**
- * Pagination component
+ * Produces the pagination number links
+ *
+ * @param {jsx} children
+ * @param {bool} active
+ * @param {func} onClick
+ *
+ * @return {jsx}
  */
-class PaginationLinks extends Component {
-  /**
-   * @constructor
-   * @param {object} props - React Component properties
-   */
-  constructor(props) {
-    super(props);
-
-    this.state = {
+function PaginationLink({
+  children,
```

Comment:
@HenriRabalais do you prefer your syntax approach here, more than the following?
```
const PaginationLink = (props) => {
   // props.children
}
PaginationLink.propTypes = {
  children: PropTypes.node,
  active: PropTypes.bool,
  onClick: PropTypes.func,
};
```

Reply:
I don't necessarily have a preference at this point. I tend to use `const ComponentName` if it's a sub component that is used exclusively by the main component contained by the file, while keeping the word `function` for components that are exported. But that's not based on any sort of standard, so I'm not tied to it.

As for propTypes, I like the idea, but I don't find them really useful from a development/testing point of view. Maybe it's just me, but I've never had useful errors thrown when I pass the wrong type of prop. Not only that, but I find they tend to be forgotten at the bottom and can be forgotten about . So I'm leaning towards declaring them in the funciton comment at the top so that the prop names, types, and defaults can all be found together. Do you have a different experience/find declaring proptypes at the bottom useful?

Reply:
@HenriRabalais I think one of @laemtl's eslint changes has a webpack error be thrown if the propsTypes aren't defined. I'm unsure if it's enabled or disabled currently. I know with syntax it's a personal preference and I'm open to being flexible. I just was curious to read your thoughts.